semantic/src/site/globals/site.overrides

```diff
@@ -2564,10 +2564,142 @@ h6:first-child {
   .hide-on-desktop {
     display: none !important;
   }
+  #menu-bar {
+    font-size: inherit;
+    .list {
+      display: flex;
+      align-items: start;
+      gap: 10px;    
+    }
+    .ui .dropdown {
+      display: flex;
+    }
+    // These selectors are so we can remove 'horizontal' prop on list in #menu-bar
+    // so that it will work with mobile 
+    .ui.list > .item:first-child,
+    .ui.list > .item:last-child {
+      padding-top: @relative3px;
+      padding-bottom: @relative3px;
+    }
+  }
 }
+
 // Mobile
 @media only screen and (max-width: @largestMobileScreen) {
   .hide-on-mobile {
     display: none !important;
   }
+
+  #content-area {
+    padding-top: calc(@mobileHeaderHeight + 12px);    
+  }
+
+  #user-area {
+    height: @mobileHeaderHeight;
+  }
+
+  #menu-bar {
+  font-size: inherit;
+    .list {      
+      z-index: 2000;
+      position: absolute;
+      top: @mobileHeaderHeight;
+      left: 0px;
+      background: @darkGrey;
+      width: 95%;
+      padding: 5px 30px 20px 25px;
+    }
+    .ui .dropdown {
+      display: flex;
+    }
+  }
+
+  /***********************
+      Mobile Hamburger
+  ***********************/
+  .hamburger {
+		width: 25px;
+		height: 25px;
+		position: absolute;
+		right: 15px;
+		top: 15px;
+		cursor: pointer;
+		display: block;
+	}
+	.hamburger 	span {
+		display: block;
+		height: 3px;
+		background: @userAreaText;
+		margin: 0 0 3px;
+	}
+	.hamburger:hover {
+		-moz-opacity: 1;
+	    -khtml-opacity: 1;
+	    -webkit-opacity: 1;
+	    opacity: 1;
+	    -ms-filter: progid:DXImageTransform.Microsoft.Alpha(opacity=100);
+	    filter: alpha(opacity=100);
+	    -moz-transition-duration: .2s;
+	    -webkit-transition-duration: .2s;
+	    -o-transition-duration: .2s;
+	    transition-duration: .2s;
+	}
+	.hamburger.active {
+				
+	}
+	.hamburger.active .hamburger-bun {
+		margin: 0;
+		-webkit-transform-origin: center center;
+		-moz-transform-origin: center center;
+		-o-transform-origin: center center;
+		transform-origin: center center;
+	}
+			
+	.hamburger.active .hamburger-bun:first-child {
+		-webkit-transform: rotate(45deg);
+	    -moz-transform: rotate(45deg);
+	    -o-transform: rotate(45deg);
+	    -ms-transform: rotate(45deg);
+	    transform: rotate(45deg);
+	    -webkit-transition: all .2s ease-out;
+	    -moz-transition: all .2s ease-out;
+	    -o-transition: all .2s ease-out;
+	    transition: all .2s ease-out;
+	    -webkit-transition: all .2s ease;
+	    -moz-transition: all .2s ease;
+	    -ms-transition: all .2s ease;
+	    -o-transition: all .2s ease;
+	    transition: all .2s ease;
+	    margin-top: 3px;
```

Comment:
Do we need all the browser-specific flags? I think these properties all work with just the main version in all modern browsers, e.g. `transform: rotate(45deg);`

Reply:
Sure don't.  Done